mslib/msui/multilayers.py

```diff
@@ -259,9 +259,9 @@ def get_multilayer_common_options(self, additional_layer=None):
             crs_values[0] = list(set(crs_values[0]).intersection(values))
 
         return sorted(elevation_values[0], key=lambda x: float(x.split()[0])) if len(elevation_values) > 0 else [], \
-            sorted(init_time_values[0]) if len(init_time_values) > 0 else [], \
-            sorted(valid_time_values[0]) if len(valid_time_values) > 0 else [], \
-            sorted(crs_values[0]) if len(crs_values) > 0 else []
+               sorted(init_time_values[0]) if len(init_time_values) > 0 else [], \
```

Comment:
this indenting  was before pep8 verified, have you checked by flake8?

Reply:
No, all the files were pep8 verified only.  For this file, I don't know how it's been modified. As far as I know, I didn't touch this file. 

Shall I still move ahead with changes at my end?

Reply:
the flake8 test rules are the one I refer too. The CI checks any pull request. If it finds errors of that definition then it blocks merging.

```
~/PycharmProjects/2021/kawalpreettkaur/MSS$ flake8 mslib 
mslib/msui/multilayers.py:262:16: E127 continuation line over-indented for visual indent
mslib/msui/multilayers.py:263:16: E127 continuation line over-indented for visual indent
mslib/msui/multilayers.py:264:16: E127 continuation line over-indented for visual indent
mslib/msui/mpl_qtwidget.py:421:39: E228 missing whitespace around modulo operator
mslib/msui/mpl_qtwidget.py:1051:83: E228 missing whitespace around modulo operator
mslib/msui/mss_pyui.py:256:74: E131 continuation line unaligned for hanging indent
mslib/utils/airdata.py:60:80: F509 '...' % ... has unsupported format character 'D'
mslib/utils/airdata.py:94:19: E127 continuation line over-indented for visual indent
mslib/utils/airdata.py:95:9: E122 continuation line missing indentation or outdented
mslib/utils/airdata.py:99:121: E501 line too long (134 > 120 characters)
mslib/utils/airdata.py:101:80: E126 continuation line over-indented for hanging indent
mslib/utils/airdata.py:101:121: E501 line too long (149 > 120 characters)
mslib/utils/airdata.py:102:121: E501 line too long (136 > 120 characters)
mslib/mswms/wms.py:84:5: E303 too many blank lines (2)
mslib/mswms/wms.py:113:5: E303 too many blank lines (2)
mslib/mswms/wms.py:124:5: E303 too many blank lines (2)
mslib/mswms/wms.py:131:5: E303 too many blank lines (2)
mslib/mswms/wms.py:324:52: E127 continuation line over-indented for visual indent
mslib/mswms/wms.py:382:60: E127 continuation line over-indented for visual indent
mslib/mswms/wms.py:389:121: E501 line too long (122 > 120 characters)
mslib/mswms/gallery_builder.py:489:121: E501 line too long (121 > 120 characters)
mslib/mswms/gallery_builder.py:514:12: E127 continuation line over-indented for visual indent
mslib/mswms/gallery_builder.py:515:21: E122 continuation line missing indentation or outdented
mslib/mswms/gallery_builder.py:734:31: E131 continuation line unaligned for hanging indent
mslib/mswms/mswms.py:111:72: E225 missing whitespace around operator
mslib/mswms/mswms.py:111:76: E231 missing whitespace after ','
mslib/mswms/mswms.py:114:44: E228 missing whitespace around modulo operator
mslib/mscolab/sockets_manager.py:40:50: E127 continuation line over-indented for visual indent
mslib/mscolab/sockets_manager.py:40:121: E501 line too long (122 > 120 characters)
mslib/mscolab/server.py:81:5: E303 too many blank lines (2)
mslib/mscolab/server.py:93:5: E303 too many blank lines (2)
mslib/mscolab/server.py:100:5: E303 too many blank lines (2)
```

Reply:
The CI does output to:
https://github.com/Open-MSS/MSS/pull/1269/checks?check_run_id=3807857617

Reply:
the CI runs on any update of your PR again.

Reply:
if files are "accidently" changed you can look on the origin in https://github.com/Open-MSS/MSS ;)

Maybe it was made by an editor configuration automatically. But it needs fixed to get merged ;)

Reply:
Okay, looking into it.